src/main/drivers/accgyro/accgyro_spi_icm426xx.c

```diff
@@ -249,7 +253,11 @@ void icm426xxGyroInit(gyroDev_t *gyro)
 
     // Configure gyro Anti-Alias Filter (see section 5.3 "ANTI-ALIAS FILTER")
     const mpuSensor_e gyroModel = gyro->mpuDetectionResult.sensor;
-    aafConfig_t aafConfig = getGyroAafConfig(gyroModel, gyroConfig()->gyro_hardware_lpf);
+    aafConfig_t aafConfig = { gyroConfig()->gyro_delt, gyroConfig()->gyro_deltSqr, gyroConfig()->gyro_bitshift };
```

Comment:
- delt may be computed as sqrt(gyro_deltSqr)
- maybe store table in flash and just select index/cutoff ?

Reply:
Yes, but round(sqrtf(deltsqr)) seems to work ...

Reply:
> This PR is just for finding good AAF cutoffs. This PR will never make it into BF like this, so no review is necessary unless something is not according to documentation.

It may be worth having it, enabled by (make parameter) define.

Also, for testing, you may be fine specifying all parameters as single value. It will save a lot of typing ;-)